sdk/communication/Azure.Communication.NetworkTraversal/README.md

```diff
@@ -80,6 +80,7 @@ foreach (CommunicationIceServer iceServer in iceServers)
     }
     Console.WriteLine($"ICE Server Username: {iceServer.Username}");
     Console.WriteLine($"ICE Server Credential: {iceServer.Credential}");
+    Console.WriteLine($"Route type: {iceServer.RouteType}");
```

Comment:
Under `### Install the package` we're referring to the wrong version number, let's update that.